lib/linguist/languages.yml

```diff
@@ -2003,6 +2003,7 @@ Hack:
   codemirror_mode: php
   codemirror_mime_type: application/x-httpd-php
   extensions:
+  - ".hack"
   - ".hh"
   - ".php"
```

Comment:
if this is a deprecated extension and not working anymore anyway with hack, then remove it because we are better of having the extension assigned to plain php.

Reply:
It's still very commonly used: https://github.com/search?q=extension%3Aphp+async+await&type=Code

So we should probably not remove it in the near future.

Reply:
@jjergus We don't retroactively remove extensions due to declining popularity. If an extension was proven to have enough real-world distribution at the time it was added, that's enough for us.